packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -190,6 +199,7 @@ export class Omnibox<T> extends React.Component<IOmniboxProps<T>, IOmniboxState<
 
     private handleQueryChange = (event: React.FormEvent<HTMLInputElement>) => {
         this.setState({ query: event.currentTarget.value });
+        Utils.safeInvoke(this.props.inputProps.onChange, event);
```

Comment:
gotta make sure `inputProps` exists before you reference it here. or do the `{ inputProps = {} }` deconstruction